pysat/tests/test_ssnl_plot.py

```diff
@@ -71,3 +71,27 @@ def test_multiple_scatterplots(self):
         assert len(figs) == 2
         assert len(axes) == 3
         assert len(axes2) == 3
+
+
+class TestDeprecation():
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+        warnings.simplefilter("always")
+
+    def teardown(self):
+        """Runs after every method to clean up previous testing."""
+
+    def test_deprecation_warning_scatterplot(self):
+        """Test if scatterplot in ssnl is deprecated"""
+
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
Should avoid one-character variable for non-iterators.  Recommend changing `w` to `war`.